helidon-linker/src/main/java/io/helidon/linker/ClassDataSharing.java

```diff
@@ -342,6 +344,10 @@ private void buildCdsArchive() throws Exception {
                 execute(action, XSHARE_DUMP, XX_SHARED_ARCHIVE_FILE + archiveFile,
                         XX_SHARED_CLASS_LIST_FILE + classListFile, UTF_8_ENCODING);
             }
+            if (Constants.OS == OSType.Windows) {
+                // Try to make the archive file writable so that a second run can delete the image
+                archiveFile.toFile().setWritable(true);
```

Comment:
Was it able to be removed the second time?. Normally when I face this issue with windows is because that file or folder is being used by other process (for example you opened a file with a text editor or a directory is open with file explorer).

Reply:
No, this didn't actually change the permission. I'm not sure if this is a problem specific to the VM or to Windows generally. This is not a huge deal because `mvn clean` works.

I can't actually run the tests on the VM either at the moment, also for a permission problem (access denied when it runs `mvn.cmd`). So I'd appreciate it if you can make sure the tests pass.

Reply:
Just saw your mail, thanks for making sure it works.